src/app/servises/model/response-data.ts

```diff
@@ -0,0 +1,23 @@
+export interface ResponseData<ResponseType> {
+    page: number,
+    total_pages: number,
+    results: ResponseType[]
+}
+
+export interface MediaResult {
+    id: number,
+    original_title: string,
+    overview: string,
+    poster_path: string,
+    vote_average: number
+}
```

Comment:
This interface should live inside 'Media' feature close to service which use this interface.

Reply:
Add Media module